test/controllers/deploys_controller_test.rb

```diff
@@ -216,6 +215,90 @@
     unauthorized :delete, :destroy, project_id: :foo, id: 1
   end
 
+  as_a_viewer do
+    before do
+      Deploy.delete_all
+      Job.delete_all
+      cmd = 'cap staging deploy'
+      project = Project.first
+      job_def =  {project_id: project.id, command: cmd, status: nil, user_id: admin.id}
+      status = [
+        {status: 'failed', production: true },
+        {status: 'running', production: true},
+        {status:'succeeded', production: true},
+        {status:'succeeded', production: false}
+      ]
+
+      status.each do |stat|
+        job_def[:status] = stat[:status]
+        job = Job.create!(job_def)
+        Deploy.create!( {
+          stage_id: Stage.find_by_production(stat[:production]).id,
+          reference: 'reference',
+          job_id: job.id
+        } )
+      end
+    end
+
+    after do
+      Deploy.delete_all
+    end
+
+    describe "finds all deploys for a deployer" do
```

Comment:
this describe can go away ... not all of these tests find all deploys for a depoyer